### PR TITLE
{bio}[foss/2021a] PyDamage v0.70 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by SIB-software**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PyDamage/PyDamage-0.70-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake',    '3.20.1'),
+    ('make',     '4.3'),
+    ('Check',    '0.15.2'),
+    ('SWIG',     '4.0.2'),
+    ('expat',    '2.2.9'),
+    ('bzip2',    '1.0.8'),
+    ('zlib',     '1.2.11'),
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyDamage/PyDamage-0.70-foss-2021a.eb:32]: @smoretti Please only include what's actually required (probably none of these in this case)

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/PyDamage/PyDamage-0.70-foss-2021a.eb (hunk 1 above)
Comment (smoretti) [reply easybuild/easyconfigs/p/PyDamage/PyDamage-0.70-foss-2021a.eb:32]: Right

File: easybuild/easyconfigs/p/PyDamage/PyDamage-0.70-foss-2021a.eb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+]
+
+dependencies = [
+    ('Python',      '3.9.5'),
+    ('Biopython',   '1.79'),
+    ('Pysam',       '0.16.0.1'),
+    ('statsmodels', '0.12.2'),
+    ('matplotlib',  '3.4.2'),
+    ('tqdm',        '4.61.2'),
```
[the remark this answers is not in the document]
Comment (smoretti) [reply easybuild/easyconfigs/p/PyDamage/PyDamage-0.70-foss-2021a.eb:41]: It does not hurt me visually like this ;-)

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3135.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/f3cf7a109072f061c6ec09e5223474d2 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @SIB-software!

**Pull request merged**